app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java

```diff
@@ -74,6 +77,15 @@ public boolean onOptionsItemSelected(MenuItem item) {
             case android.R.id.home:
                 finish();
                 return true;
+            case R.id.share_speakers_url:
+                Intent sendIntent = new Intent();
+                sendIntent.setAction(Intent.ACTION_SEND);
+                sendIntent.putExtra(Intent.EXTRA_SUBJECT, "Speaker profile");
+                sendIntent.putExtra(Intent.EXTRA_TEXT, selectedSpeaker.getName() + " is speaking at (event name goes here) on the topic(topic goes here)." +
```

Comment:
Could you not send this hard coded string. Please add the event name from the app name, topics from the arraylist of topics by speaker. Also for the app link, add a constant that can be changed from one place and used at other places as well.